backend/src/main/scala/com/softwaremill/adopttapir/starter/formatting/ProjectFormatter.scala

```diff
@@ -0,0 +1,51 @@
+package com.softwaremill.adopttapir.starter.formatting
+
+import cats.effect.IO
+import com.softwaremill.adopttapir.logging.FLogging
+import com.softwaremill.adopttapir.starter.files.FilesManager
+import com.softwaremill.adopttapir.template.{CommonObjectTemplate, GeneratedFile}
+import org.scalafmt.interfaces.Scalafmt
+
+import java.nio.file.Path
+
+class ProjectFormatter(fm: FilesManager) extends FLogging {
```

Comment:
nit: naming AFAIU it is used to format files in the project therefore IMHO `FilesFormatter` would better reflect its purpose

Reply:
It's a bit more complicated. We format something called `GeneratedFile` - entities that consist of `path` and `content` strings. Renaming `ProjectFormatter` to `FilesFormatter` implies that `File`s are involved and that is not true. Perhaps `GeneratedFilesFormatter` would be a better name, but it is quite long name...

Reply:
+1 to `GeneratedFilesFormatter`

Reply:
Renamed then.